build.xml

```diff
@@ -811,6 +811,16 @@ To get started using Eclipse, execute "./build.py build-dev" and import the top-
             </copy>
     </target>
 
+    <target name="copy-bftools" depends="init">
+            <copy todir="${dist.dir}/bin">
+                <fileset dir="${bioformats.comp}/tools"/>
+            </copy>
+            <chmod dir="${dist.dir}/bin" perm="a+x" includes="*"/>
+            <copy todir="${dist.dir}/bin">
+                <fileset file="${bioformats.comp}/artifacts/bioformats_package.jar"/>
```

Comment:
Yup, had considered that. Would certainly save some space. That being said, it might be that having package.jar in with the clients will get more testing for it.